device/arista/x86_64-arista_7050_qx32s/Arista-7050QX-32S-S4Q31/hwsku.json

```diff
@@ -1,143 +1,131 @@
 {
     "interfaces": {
         "Ethernet0": {
-            "default_brkout_mode": "1x10G",
-            "port_type": "RJ45"
-        },
-        "Ethernet1": {
-            "default_brkout_mode": "1x10G",
-            "port_type": "RJ45"
-        },
-        "Ethernet2": {
-            "default_brkout_mode": "1x10G",
-            "port_type": "RJ45"
-        },
-        "Ethernet3": {
-            "default_brkout_mode": "1x1G",
+            "default_brkout_mode": "4x10G[1G]",
```

Comment:
@byu343 Please make change as below so that we can support 3x10G + 1x1G

```
        "Ethernet0": {
            "default_brkout_mode": "3x10G(3)+1x1G(1)",
            "port_type": "RJ45"
        },
```